### DISCUSSION TITLE
Incomplete Alerts

### DISCUSSION THREAD

**Discussion by BrentMButler** [weather-gov/api#517] [Q&A]
The detailed text description of an alert is more detailed and descriptive online than what is found in the API response.

Casual visitor going to weather.gov would find (https://forecast.weather.gov/wwamap/wwatxtget.php?cwa=dmx&wwa=winter%20storm%20warning):

> URGENT - WINTER WEATHER MESSAGE
> National Weather Service Des Moines IA
> 259 PM CST Thu Jan 13 2022
> 
> ...Winter Storm to Impact the Area Friday through Early Saturday
> Morning...
> 
> .A winter storm will impact much of central Iowa Friday into
> early Saturday bringing moderate to heavy snow to much of the
> area. Travel impacts are expected, especially the afternoon school
> departure and the evening commute. 4-10 inches of snow is
> expected across the area with locally higher amounts possible. The
> winds will be out of the east/northeast and will increase to 15
> to 25 mph with a few gusts over 30 mph by Friday night. The wind
> will produce pockets of blowing and drifting snow, particularly in
> open and rural areas, during the afternoon and evening.
> 
> IAZ004>007-015>017-023>028-033>039-140500-
> /O.UPG.KDMX.WS.A.0001.220114T1500Z-220115T1200Z/
> /O.NEW.KDMX.WS.W.0001.220114T1200Z-220115T0900Z/
> Emmet-Kossuth-Winnebago-Worth-Palo Alto-Hancock-Cerro Gordo-
> Pocahontas-Humboldt-Wright-Franklin-Butler-Bremer-Sac-Calhoun-
> Webster-Hamilton-Hardin-Grundy-Black Hawk-
> Including the cities of Estherville, Algona, Forest City,
> Lake Mills, Northwood, Manly, Emmetsburg, Garner, Britt, Kanawha,
> Mason City, Clear Lake, Pocahontas, Laurens, Rolfe, Fonda,
> Gilmore City, Humboldt, Eagle Grove, Clarion, Belmond, Hampton,
> Parkersburg, Clarksville, Shell Rock, Greene, Aplington, Allison,
> Dumont, Waverly, Sac City, Lake View, Odebolt, Wall Lake,
> Schaller, Early, Rockwell City, Manson, Lake City, Pomeroy,
> Fort Dodge, Webster City, Iowa Falls, Eldora, Ackley,
> Grundy Center, Reinbeck, Conrad, Dike, Wellsburg, Waterloo,
> and Cedar Falls
> 259 PM CST Thu Jan 13 2022
> 
> ...WINTER STORM WARNING IN EFFECT FROM 6 AM FRIDAY TO 3 AM CST
> SATURDAY...
> 
> * WHAT...Heavy snow expected. Total snow accumulations of 6 to 10
>   inches.
> 
> * WHERE...Northern into Central Iowa.
> 
> * WHEN...From 6 AM Friday to 3 AM CST Saturday.
> 
> * IMPACTS...Travel is expected to be difficult. Patchy blowing
>   snow could significantly reduce visibility. The hazardous
>   conditions are expected to impact the evening commute.
> 
> PRECAUTIONARY/PREPAREDNESS ACTIONS...
> 
> If you must travel, keep an extra flashlight, food, and water in
> your vehicle in case of an emergency.
> 
> For the latest travel conditions, check the Iowa 5 1 1 app or
> dial 5 1 1.
> 
> &&
> 
> $$
> 

The API call doesn't show any of that opening paragraph (https://api.weather.gov/alerts/urn:oid:2.49.0.1.840.0.f7ea1d1d8ebe943c9bac0ae1adc559d2c5903319.002.2)

> headline: "Winter Storm Warning issued January 13 at 2:59PM CST until January 15 at 6:00AM CST by NWS Des Moines IA",
> description: "* WHAT...Heavy snow expected. Total snow accumulations of 6 to 10
> inches.
> 
> * WHERE...Much of Central Iowa.
> 
> * WHEN...From 9 AM Friday to 6 AM CST Saturday.
> 
> * IMPACTS...Travel is expected to be difficult. Patchy blowing
> snow could significantly reduce visibility. The hazardous
> conditions are expected to impact the evening commute.",
> instruction: "If you must travel, keep an extra flashlight, food, and water in
> your vehicle in case of an emergency.
> 
> For the latest travel conditions, check the Iowa 5 1 1 app or
> dial 5 1 1."

Is this just not included in the same alert record perhaps?

**Comment by oisact** [comment]
This source of WNS alerts doesn't include that additional text either. There must be an additional text field that local weather offices can use that isn't part of the actual alert. I don't believe the alert itself, as heard on NWS weather radio, TV, emergency broadcast, etc, contains that additional text either.
https://alerts.weather.gov/cap/wwacapget.php?x=IA1263DCB79740.WinterStormWarning.1263DCC859E0IA.DVNWSWDVN.44c321217a586bdfb84eebd73b03b1ce

**Comment by jah1007** [comment]
The upstream source of the data does not include the text. They are a wide area synopsis and do not provide targeted warning information which is why they are ignored when building the data upstream.
